activeadmin.gemspec

```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'railties',            '>= 4.0', '< 5.1'
   s.add_dependency 'ransack',             '~> 1.3'
   s.add_dependency 'sass-rails'
+  s.add_dependency 'thor',                '0.19.1', '!=0.19.2', '!=0.19.3', '!=0.19.4' # http://stackoverflow.com/questions/41207432/
```

Comment:
`'0.19.1', '!=0.19.2', '!=0.19.3', '!=0.19.4'` doesn't make sense.
`'~>0.19.1', '!=0.19.2', '!=0.19.3', '!=0.19.4'`, would make sense.
Be in general I think we should not fix third party bugs. We only should limiting version if our code has a problem with this version, not if JBuilder has a problem with this version.